WeHere/Managers/SNSLoginManager.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SNSLoginManager.swift
+//  WeHere
+//
+//  Created by Inho Choi on 2022/11/20.
+//
+
+import SwiftUI
+import KakaoSDKCommon
+import KakaoSDKAuth
+import KakaoSDKUser
+
+class SNSLoginManager {
+    private init() { }
+    
+    static let shared = SNSLoginManager()
+    
+    func signInWithKakao() {
+        UserApi.shared.loginWithKakaoAccount {(oauthToken, error) in
+            if let error = error {
+                print(error)
+            }
+            else {
+                print("loginWithKakaoAccount() success.")
+                
+                //do something
+                _ = oauthToken
+            }
+        }
+    }
+    
+    
+}
```

Comment:
p2)
린트에 엄청걸려서 수정해둬야할것같습니다!

```suggestion
    func signInWithKakao() {
        UserApi.shared.loginWithKakaoAccount {(oauthToken, error) in
            if let error = error {
                print(error)
            } else {
                print("loginWithKakaoAccount() success.")
                
                // TODO: token 관련 작업
                _ = oauthToken
            }
        }
    }
}

```

Reply:
카카오 로직을 수정했습니다. 지금은 View에 속해있는데 View 다 합치고 난 이후에 ViewModel 분리하겠습니다.